src/domains/ioweb-common/06_cdn_itn.tf

```diff
@@ -47,3 +52,159 @@ module "io_web_profile_itn_fe_st" {
 
   tags = var.tags
 }
+
+resource "azurerm_cdn_frontdoor_profile" "portal_profile" {
+  name                = format("%s-profile-portal-afd-01", local.project_itn)
```

Comment:
I suggest to name the resource without the location (`io-p-profile-portal-afd-01`) as it is global

Reply:
renamed in https://github.com/pagopa/io-infra/pull/1341/commits/4cad3e946adfbaa1ee253b1c0bf10446529de13d